### DISCUSSION TITLE
Any way to only trigger keycommand if element is visible?

### DISCUSSION THREAD

**Discussion by seabasss** [danharrin/alpine-mousetrap#4] [Ideas]
Is there a way to only trigger the shortcut if the element it's on is visible?

And is there a way to prioritize the shortcuts if there are multiple ones with the same command on the screen?
Ex. I trigger a button click with ctrl+n which opens a modal and inside the modal is another button that also has the same key command that should now be prioritized until the modal is closed? Like a keycommand trap basically.

Thank you!

**Comment by danharrin** [comment]
Hmm I'm not sure, is it supported in the underlying mousetrap library?
